pages/common/xml.md

```diff
@@ -0,0 +1,29 @@
+# xml
+
+> XMLStarlet Toolkit: Query, edit, check, convert and transform XML documents.
+> Subcommands: `canonic`, `edit`, `elements`, `escape`, `unescape`, `format`, `list`, `pyx`, `depyx`, `select`, `transform`, or `validate`.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Execute a subcommand with input from a file or URI and using standard output:
+
+`xml {{subcommand}} {{options}} {{path/to/input.xml|URI}}`
```

Comment:
This doesn't hold true for all subcommands though (e.g. `xml unescape`)?

Reply:
No, only for the majority of them. `escape` and `unescape` are exceptions.